app/initializers/google-drive-permission.js

```diff
@@ -1,7 +0,0 @@
-export default {
-  name: 'google-drive-permission',
-  after: 'google-drive',
-  initialize: function (container, application) {
-    application.inject('adapter:google-drive-permission', 'documentSource', 'document-source:main');
-  }
-};
```

Comment:
This is one of the initializers that got moved into the new `document-source` initializer.